frontend/src/routes/manage/Video/TechnicalDetails.tsx

```diff
@@ -115,35 +117,56 @@ export const TrackInfo: React.FC<TrackInfoProps> = (
         tracks.push({ resolution, mimetype, uri });
     }
 
+    const isSingleStream = flavors.size === 1;
+
     return <section css={className}>
         <h2>{t("manage.my-videos.technical-details.tracks")}</h2>
         <ul css={{ fontSize: 15, marginTop: 8, paddingLeft: 24 }}>
             {Array.from(flavors, ([flavor, tracks]) => <li key={flavor}>
-                {translateFlavors ? flavorTranslation(flavor) : <code>{flavor}</code>}
-                <ul>{tracks
+                {!isSingleStream && (translateFlavors
+                    ? flavorTranslation(flavor)
+                    : <code>{flavor}</code>)
+                }
+                <Tracks {...{ isSingleStream }}>{tracks
                     .sort((a, b) => (a.resolution?.[0] ?? 0) - (b.resolution?.[0] ?? 0))
                     .map((track, i) => <TrackItem key={i} {...track} />)
-                }</ul>
+                }</Tracks>
             </li>)}
         </ul>
     </section>;
 };
 
+type TracksProps = PropsWithChildren<{ isSingleStream: boolean }>
+const Tracks: React.FC<TracksProps> = ({ isSingleStream, children }) =>
+    isSingleStream ? <>{children}</> : <ul>{children}</ul>;
+
 const TrackItem: React.FC<SingleTrackInfo> = ({ mimetype, resolution, uri }) => {
     const { t } = useTranslation();
+    const type = mimetype && mimetype.split("/")[0] as "audio" | "video";
```

Comment:
Mhhh I'm not super comfortable with that type cast? Because we have no guarantee that this string is actually always either "audio" or "video". By casting it like this, the "something else" case is swept under the rug and Typescript doesn't remind us of it. We should somehow deal with this being something other than audio/video.

Reply:
Ye, so what I have in mind is sth like

```tsx
(type === "audio" || type === "video") ?  t(`translationkey.${type}`) : type
```

Or something like that. It will be very rare, so it doesn't need to look super pretty in that case.

Reply:
ah sure that works for me